stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -126,7 +126,12 @@ function stride(a::Union{DenseArray,StridedReshapedArray,StridedReinterpretArray
     end
     return s
 end
-strides(a::Union{DenseArray,StridedReshapedArray,StridedReinterpretArray}) = size_to_strides(1, size(a)...)
+strides(a::Array) = size_to_strides(1, size(a)...)
+strides(a::BitArray) = size_to_strides(1, size(a)...) #TODO: This is needed for permutedims!, but its not clear what stride means here
```

Comment:
Is this still needed?  I would guess that it got fixed by removing `BitArray` from the `DenseArray` abstract type.

Reply:
Ah, because DenseArray didn't use to mean contiguous in memory... but now it does #26085

 (I'm not sure if the other place I made this comment worked...)

Reply:
Bump